extensions/python/src/main/java/com/hazelcast/jet/python/PythonServiceContext.java

```diff
@@ -133,8 +132,6 @@ void destroy() {
             }
         } catch (Exception e) {
             throw new JetException("PythonService cleanup failed: " + e, e);
-        } finally {
-            IOUtil.delete(runtimeBaseDir);
```

Comment:
Why remove this?

Reply:
We delete the directory passed to us which contains the init script and the actual python handler code. This directory is created by the user prior to submitting the job, user may have other resources in the directory or user may want to re-use the job related scripts for further use. I think it was wrong to delete this directory.

Reply:
Ok, but let's wait for Marko's 5 cents too, because I wouldn't be surprised if he had a good reason for adding this delete.

Reply:
Yes, this deletes the temp directory created in the `PythonServiceContext` constructor, the `context.attachedDirectory` call. However, `ProcSupplierCtx` remembers for which atttached directories it already created a temp directory, so a repeated call to `attachedDirectory` won't repeat this.